### PR TITLE
Terraform v0.14 update

### PR DESCRIPTION
**Pull request by robmorgan**
This PR updates the repo to support Terraform 0.14.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -7,11 +7,12 @@ env: &env
   environment:
     GRUNTWORK_INSTALLER_VERSION: v0.0.30
     TERRATEST_LOG_PARSER_VERSION: v0.30.4
-    MODULE_CI_VERSION: v0.29.1
-    TERRAFORM_VERSION: 0.13.5
+    MODULE_CI_VERSION: v0.31.0
```
Comment (marinalimeira) [inline .circleci/config.yml:10]: The latest for module ci is `v0.33.2`, and the release with TF14 is `v0.32.0`

Comment (robmorgan) [reply .circleci/config.yml:10]: @marinalimeira oh bummer, maybe I used the wrong version of the script: `tf14.rb`. Let me check

Comment (robmorgan) [reply .circleci/config.yml:10]: @marinalimeira should I update this? https://github.com/gruntwork-io/prototypes/blob/master/git-xargs-assets/scripts/tf14.rb#L27

Comment (robmorgan) [inline .circleci/config.yml:10]:
```suggestion
    MODULE_CI_VERSION: v0.33.2
```

Comment (robmorgan) [reply .circleci/config.yml]: Changed to v0.33.2

**Review by marinalimeira — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (robmorgan) [reply .circleci/config.yml]: Changed to v0.33.2

**Pull request merged**